metis-authentication/metis-authentication-common/src/main/java/eu/europeana/metis/authentication/user/MetisUserRecord.java

```diff
@@ -0,0 +1,178 @@
+package eu.europeana.metis.authentication.user;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+/**
+ * The Metis user containing all parameters.
+ * <p>This class is used as a JPA class to the postgresql database</p>
+ *
+ * @author Simon Tzanakis (Simon.Tzanakis@europeana.eu)
+ * @since 2017-10-27
+ */
+@Entity
+@Table(name = "metis_users")
+public class MetisUserRecord {
```

Comment:
If we are going for this approach, to splitting model classes and representation classes. We should agree on naming suffices.. I wouldn't vote for the current suffix of "Record"..

Reply:
After discussion, I changed the name of this class to MetisUserModel.

Reply:
Great